auditbeat/Makefile

```diff
@@ -50,8 +50,9 @@ fields: python-env
 # Collects all module configs
 .PHONY: configs
 configs: python-env
-	@cat ${ES_BEATS}/auditbeat/_meta/common.yml \
-	     <(go run scripts/generate_config.go -os linux -concat) > _meta/beat.yml
+	@cat ${ES_BEATS}/auditbeat/_meta/common.p1.yml \
```

Comment:
This seems to also contain the other packaging fix 👍

Reply:
It doesn't, but I opened a backport PR for that one as well: https://github.com/elastic/beats/pull/5127